eth.js.org

WoeOm opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 commented on cnames_active.js:
```diff
@@ -568,6 +568,7 @@ var cnames_active = {
   "escape": "licshee.github.io/Escape.js", // noCF? (don´t add this in a new PR)
   "esfx": "esfx.github.io",
   "esper": "codecombat.github.io/esper.js",
+  "eth": "https://eth-js.github.io/eth-dev-tools/#/",
```
```suggestion
  "eth": "eth-js.github.io/eth-dev-tools",
```
Please use the correct target format like everyone else.

WoeOm replied:
updated

WoeOm commented:
> Hi there!
> 
> Before we can accept your domain request, you need to create a file named `CNAME` in the root of your github pages site (_this may not be the root of your repository_). 📝
> This file should contain a single line with the full js.org domain you are requesting, in this instance `eth.js.org`.

updated

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to to fix the path issues of your page so I can have a look.
I'll merge after we have "contact".

WoeOm commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to to fix the path issues of your page so I can have a look.
> I'll merge after we have "contact".

I have fixed the paths.

indus commented:
Welcome https://eth.js.org
